default/web_tt2/news.tt2

```diff
@@ -3,53 +3,78 @@
-   [% IF list_to_admin.value.mod_signoff %]<a href="[% 'sigindex' | url_rel([list_to_admin.key]) %]">[%|loc(list_to_admin.value.mod_signoff)%]%1 unsubscription request(s)[%END%]</a>[% END %]
-  </li>
- [% END -%]
-</ul>
-</div>
-[% ELSE %]
-<br>
-<!-- do nothing-->
-[% END %]
-
-<!-- list of new errors-->
-[% IF errors_summary.size -%]
-<h2>[%|loc%]User Errors[%END%]</h2>
-<p>[%|loc%]We received error reports while trying to send you messages. Some of them may have been lost.[%END%]</p>
-<div id="home_admin_summary">
-<ul>
-[% FOREACH user_with_errors = errors_summary -%]
-<li>[%|loc(user_with_errors.key)%] List %1: [%END%]
-[% IF user_with_errors.value %] [%|loc(user_with_errors.value.count)%] %1 error(s)[% END %], 
-[% | mailto("${user_with_errors.key}-request@${domain}") ~%]
-  [%|loc()%]please contact the list owners[% END %]
-[%~ END%]
-<ul>
-<li>[%|loc(user_with_errors.value.first)%]first bounce: %1[% END %]</li>
-<li>[%|loc(user_with_errors.value.last)%]last bounce: %1[% END %]</li>
-<li>[%|loc(user_with_errors.value.type)%]Last bounce category: %1[% END %]</li>
-[% END %]<!--IF-->
-</li>
-[% END %]<!--FOREACH-->
-</ul>
-
-</div>
-[% END %]<!--IF-->
-</div>
+        <!-- list of new errors-->
+        [% IF errors_summary.size -%]
+            <h2>[%|loc%]User Errors[%END%]</h2>
+            <p>[%|loc%]We received error reports while trying to send you messages. Some of them may have been lost.[%END%]</p>
+            <div id="home_admin_summary">
+                <ul>
+                    [% FOREACH user_with_errors = errors_summary -%]
+                        <li>
+                            [%|loc(user_with_errors.key)%] List %1: [%END%]
+                            [% IF user_with_errors.value %]
+                                [%|loc(user_with_errors.value.count)%] %1 error(s)[% END %],
+                                [% | mailto("${user_with_errors.key}-request@${domain}") ~%]
+                                    [%|loc()%]please contact the list owners[% END %]
+                                [%~ END%]
+                                <ul>
+                                    <li>
+                                        [%|loc(user_with_errors.value.first)%]first bounce: %1[% END %]
+                                    </li>
+                                    <li>
+                                        [%|loc(user_with_errors.value.last)%]last bounce: %1[% END %]
+                                    </li>
+                                    <li>
+                                        [%|loc(user_with_errors.value.type)%]Last bounce category: %1[% END %]
+                                    </li>
+                                </ul>
+                            [% END %]<!--IF-->
+                        </li>
+                    [% END %]<!--FOREACH-->
+                </ul>
+            </div>
+        [% END %]<!--IF-->
```

Comment:
`[% END # IF %]` and `[% END # FOREACH %]` are better:
``` code
                            [% END # IF %]
                        </li>
                    [% END # FOREACH %]
                </ul>
            </div>
        [% END # IF %]
```

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.